### DISCUSSION TITLE
Feature Request: Adobe Web SDK - Add filters for specific event call types

### DISCUSSION THREAD

**Discussion by frank-8899** [MisterPhilip/omnibug#277] [Ideas]
When debugging the Web SDK in Omnibug I see calls for various Web SDK event calls:
web.webPageDetails.pageViews
web.webinteraction.linkClicks 
web.webInteraction.targetLoad
decisioning.propositionDisplay
decisioning.propositionFetch
there might be more as well

It would be great if, using the Filter Requests section, I could filter out the ones I don't want, so my Omnibug view is less cluttered. Thanks

**Comment by MisterPhilip** [comment]
Agreed, it can get overwhelming and hard to find what you're looking for. This is planned, I just need to find some time to work on Omnibug again :(
